web/urls.py

```diff
@@ -133,7 +133,6 @@
         "task/<int:task_id>/csv", teacher_view.download_csv_per_task, name="download_csv_per_task"
     ),
     path("tasks", teacher_view.all_tasks, name="tasks"),
-    path("tasks/<str:subject__abbr>", teacher_view.all_tasks, name="tasks"),
```

Comment:
I wonder if someone used this :) Didn't find it in code though.

Reply:
It was used on old page with all tasks, when teacher choosed subject by clicking at link with specific subject abbr, then he was redirected to for example /tasks/UPR

Reply:
https://github.com/mrlvsb/kelvin/blob/master/templates/web/all_tasks.html#L5

Reply:
Ok, in that case we can remove it now.